### DISCUSSION TITLE
Deploy 11ty on Google Buckets, css and navigation links not working.

### DISCUSSION THREAD

**Discussion by rahianeja** [11ty/eleventy#1690] [Q&A]
Is it possible to deploy static files generated on google bucket? When I build my project using with `NODE_ENV=production` and then `eleventy` commands, it generates the files in dist folder.
In this dist folder, when I try to access index.html on local machine, neighther css nor navigation works. Same on google buckets, after copying files on buckets css and other relative links doesn't work. Can anyone help me how I can get this app running without doing npm run serve.

**Comment by pdehaan** [comment]
I haven’t tried deploying to a google bucket, but how are you testing it locally if you aren’t using `npm run serve`? 
If you’re trying to test the site in your browser using C:\Users\xyz, it probably won’t render as expected.

Unless Google buckets deploy to a root directory, you might need to configure your `pathPrefix`.
https://www.11ty.dev/docs/config/#deploy-to-a-subdirectory-with-a-path-prefix
I’ve tried deploying to GitHub Pages before and that’s tripped me up a few times.
